var/spack/repos/builtin/packages/moab/package.py

```diff
@@ -32,8 +33,6 @@ class Moab(AutotoolsPackage):
     version("5.2.1", sha256="60d31762be3f0e5c89416c764e844ec88dac294169b59a5ead3c316b50f85c29")
     version("5.2.0", sha256="805ed3546deff39e076be4d1f68aba1cf0dda8c34ce43e1fc179d1aff57c5d5d")
     version("5.1.0", sha256="0371fc25d2594589e95700739f01614f097b6157fb6023ed8995e582726ca658")
-    version("5.0.2", commit="01d05b1805236ef44da36f67eb2701095f2e33c7")
-    version("5.0.1", commit="6cc12bd4ae3fa7c9ad81c595e4d38fa84f0884be")
```

Comment:
Thanks for the quick response @tldahlgren. Well, the reason for the removal of those two versions is that I did not find the tarballs for those versions in the FTP site that was there originally. These are not available on the new site either. 

The [MOAB tags page](https://bitbucket.org/fathomteam/moab/downloads/?tab=tags) does not list these versions. So if someone tries using these 6-year-old versions, it will fail. I'd rather avoid it instead of deprecating an unavailable version. Thoughts?

Reply:
We usually recommend deprecating the versions until the next release in case someone has those versions in a build cache somewhere.  But they are minor and, as you say, they're old so I am fine merging as is or after adding deprecation to the versions.  You're the maintainer so let me know what you want.

Reply:
Understood. I agree that removing versions should be avoided in general (and should be deprecated first). However, I do not have the tarball accessible for these versions to update the download sites; so would rather remove it from the recipe altogether to avoid confusion. If someone still requires the use of MOAB 5.0.1 or 5.0.2, I would recommend they update to a more recent version anyway. If any issues arise, I'm happy to address user concerns.